docs/data-routing/3-transformers/10-location.md

```diff
@@ -1,29 +1,62 @@
+points. Data must be formatted as described below, and at least one cell tower
+or Wi-Fi access point must be provided for successful resolution.
 
 | Attribute | Description                                                  |
 | --------- | ------------------------------------------------------------ |
 | `cell`      | Array of cell tower objects. |
 | `wifi`  | Array of Wi-Fi access point objects.                  |
 
+#### Cell Tower Format
+
+| Attribute  | Description                                                                        | Required |
+|------------|------------------------------------------------------------------------------------|----------|
+| `type`     | String indicating type of cellular network (`ltecatm` or `nbiot`                   | X        |
+| `mcc`      | Integer indicating Mobile Country Code.                                            | X        |
+| `mnc`      | Integer indicating Mobile Network Code.                                            | X        |
+| `id`       | Integer indicating EUCID (LTE Cat-M) or Cell ID (NB-Iot).                          | X        |
+| `strength` | Integer indicating RSRP (LTE Cat-M) or NRSRP (NB-Iot). (dBm)                            |          |
+| `lac`      | Integer indicating Tracking Area Code (TAC)                                        |          |
+| `age`      | Integer indicating age of measurement. (ms)                                             |          |
+| `channel`  | Integer indicating EARFCN.                                                         |          |
+| `serving`  | Boolean indicate whether device is currently served by the cell. (default: `false`) |          |
+| `lid`      | Integer indicating PCI (LTE Cat-M) or NCID (NB-Iot)                                |          |
+
+#### Wi-Fi Access Point Format
+
+| Attribute   | Description                                                                                      | Required |
+|-------------|--------------------------------------------------------------------------------------------------|----------|
+| `mac`       | String indicating access point MAC address.                                                      | X        |
+| `ssid`      | String indicating access point Service Set Identifier.                                           |          |
+| `strength`  | Integer indicating signal strength of access point (dBm).                                        |          |
+| `age`       | Integer indicating age of measurement. (ms)                                                      |          |
+| `frequency` | Integer indicating frequency of access point. (MHz)                                              |          |
+| `channel`   | Integer indicating channel number of access point.                                               |          |
+| `Connected` | Boolean indicating whether device is currently connected to the access point. (default: `false`) |          |
+
```

Comment:
Do we want to add any info about what fields are required and what are optional? I ask because the example do not include all fields.

I'm also wondering if there should be more clarification about `age`. Is this ms between the reading and submitting them to Golioth or is this an epoch?

Reply:
> Do we want to add any info about what fields are required and what are optional?

@szczys the last column indicates whether the field is required or not -- are you thinking we could restructure to make that more clear?

You also bring up a good point that it is probably worth at least adding a note that `net_info` support in the SDK can craft this for you.

> I'm also wondering if there should be more clarification about age. Is this ms between the reading and submitting them to Golioth or is this an epoch?

`age` is the relative age of the measurement, so it would be more akin to the former. I can add a note to clarify 👍🏻